kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/AbstractDockerMojo.java

```diff
@@ -648,5 +649,20 @@ protected GavLabel getGavLabel() {
         // Label used for this run
         return new GavLabel(project.getGroupId(), project.getArtifactId(), project.getVersion());
     }
-
+    
+    public ResourceConfig getResources() {
+        return this.resources;
+    }
```

Comment:
Umm, why do we need getters for these fields? These fields are `protected` so they should be  visible in classes extending this class.

Reply:
You are right it should be, however in my IDE I got error about visibility, thereforeI decided to use getters/setters methods. However let me run a single test without gettrs/setter and let's see :)

Reply:
Tried to run a test without getters, got compilation error:
> $ cd kubernetes-maven-plugin/plugin
$ mvn test -Dtest=WatchMojoTest
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/arsenal/eclipse-workspace/jkube/kubernetes-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/develop/WatchMojoTest.java:[123,14] resources has protected access in org.eclipse.jkube.maven.plugin.mojo.build.AbstractDockerMojo
[ERROR] /home/arsenal/eclipse-workspace/jkube/kubernetes-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/develop/WatchMojoTest.java:[135,14] namespace has protected access in org.eclipse.jkube.maven.plugin.mojo.build.AbstractDockerMojo
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  18.900 s
[INFO] Finished at: 2024-02-26T09:50:56+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:testCompile (default-testCompile) on project kubernetes-maven-plugin: Compilation failure: Compilation failure: 
[ERROR] /home/arsenal/eclipse-workspace/jkube/kubernetes-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/develop/WatchMojoTest.java:[123,14] resources has protected access in org.eclipse.jkube.maven.plugin.mojo.build.AbstractDockerMojo
[ERROR] /home/arsenal/eclipse-workspace/jkube/kubernetes-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/develop/WatchMojoTest.java:[135,14] namespace has protected access in org.eclipse.jkube.maven.plugin.mojo.build.AbstractDockerMojo

Reply:
Yes, sure! However what was the issue?

Reply:
Sorry, I was trying a different approach of not relying on getters/setters at all, but as per https://github.com/eclipse/jkube/pull/2729#discussion_r1503844358 it makes the test slightly more difficult to maintain. I've reverted it to what you had done earlier.

I tried removing getters but I didn't get the abovementioned error.